rita_common/src/payment_controller/mod.rs

```diff
@@ -153,7 +153,7 @@ async fn make_payment(mut pmt: PaymentTx) -> Result<(), PaymentControllerError>
         "current balance: {:?}, payment of {:?}, from address {} to address {} with nonce {}",
         balance, pmt.amount, our_address, pmt.to.eth_address, nonce
     );
-    if balance < pmt.amount {
+    if balance.clone().unwrap_or_else(|| 0u64.into()) < pmt.amount {
```

Comment:
handle the balcne is none case above this, just return the same payment controller error with a different log statement. What you have here is correct in that it will do the right thing but handling the bootstrapping case explicitly is more clear and lets us log that the node is stuck in the bootstrapping case which is important info if a bug happens there.